lib/rack/access_log.rb

```diff
@@ -28,7 +28,8 @@ def create_log_message_base(request)
         remote_ip: request.ip || '-',
         request_path: request.path_info,
         query_string: request.query_string,
-        request_method: request.request_method
+        request_method: request.request_method,
+        body: request.body.string
```

Comment:
`request.body` can't be nil only an empty string, right?

Reply:
yes.